test/jdk/tools/launcher/ExecutionEnvironment.java

```diff
@@ -24,49 +24,27 @@
 /*
  * @test
  * @bug 4780570 4731671 6354700 6367077 6670965 4882974
- * @summary Checks for LD_LIBRARY_PATH and execution  on *nixes
- * @requires os.family != "windows" & !vm.musl & os.family != "aix"
```

Comment:
Hello @JoKern65, I haven't yet fully grasped the change, but I think the:
```
@requires os.family != "windows"
```
part should still be retained to prevent this test from being launched on Windows.

Reply:
Good catch. The test doesn't fail on Windows but it also doesn't make sense because it aims to test Unix specific things.